apps/services/auth-admin-api/src/app/modules/resources/resources.controller.ts

```diff
@@ -77,7 +77,7 @@ export class ResourcesController {
   @Get('api-resources')
   @ApiQuery({ name: 'page', required: true })
   @ApiQuery({ name: 'count', required: true })
-  // TODO: Figure this out: @ApiOkResponse({  type: { rows: ApiResource[]; count: number }, isArray: true })
+  @ApiOkResponse({ type: ApiResource, isArray: true })
   async findAndCountAllApiResources(
```

Comment:
FindAndCountAll always returns { rows: array of type and count: number }
In this case: type: { rows: ApiResource[]; count: number }.
You have changed this like it returns ApiResource array. Was that the goal?